src/sprites.asm

```diff
@@ -777,60 +922,321 @@ RTS
 .koopanumbers
 db $04,$04,$05,$05,$07
 
-Sprite0D:		;keyhole
+Sprite0D:;Bob-omb
+JSR Sprite0F
+RTS
+
+Sprite0E:;Key Hole
+RTS
+
+GoombaWins:
+JSR HurtLuigi
+RTS
+
+Sprite0F:;Goomba
+LDA $1570,x
+BEQ $03
+JMP KillWithStar
+LDA $14C8,y
+CMP #$09
+BNE GoombaReg
+akakakakaka:
```

Comment:
labels should generally be descriptive...

Reply:
sorry, this code is a copy of the interaction of the koopa (changed to work)